index.js

```diff
@@ -47,11 +48,22 @@ class Conf {
 		};
 
 		if (!options.cwd) {
-			if (!options.projectName) {
-				const pkgPath = pkgUp.sync(parentDir);
-				// Can't use `require` because of Webpack being annoying:
-				// https://github.com/webpack/webpack/issues/196
-				options.projectName = pkgPath && JSON.parse(fs.readFileSync(pkgPath, 'utf8')).name;
+			const packageData = this._getPackageData();
+
+			if (packageData && !options.projectName) {
+				const {name} = packageData;
+
+				if (name) {
```

Comment:
We cant simplify these conditions, we can just do:

```js
if (!options.projectName) {
   options.projectName = packageData.name;
}
```

which was like it was before.. and also:

if (!options.projectVersion) {
   options.projectVersion= packageData.version;
}

Reply:
I don't think we can, or am I missing something?

If the package.json couldn't be inferred (it happens in the tests for example) it causes the code to break (`cannot read property version of undefined`)

Reply:
Sorry, I've updates the comment above.

Reply:
Done 👍